tests/lib/rules/require-ember-lifeline.js

```diff
@@ -0,0 +1,113 @@
+const rule = require('../../../lib/rules/require-ember-lifeline');
+const MESSAGE = rule.meta.message;
+const RuleTester = require('eslint').RuleTester;
+const ruleTester = new RuleTester();
+
+ruleTester.run('require-ember-lifeline', rule, {
+  valid: [
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            this.runTask(() => {
+              doSomeWork();
+            });
+          }
+        });`,
+      parserOptions: {
```

Comment:
Can we move the `parserOptions` up into the `RuleTester()` and then remove the duplicated `parserOptions`?

example: 
```js
const ruleTester = new RuleTester({
  parserOptions: { ... }
});
```